openapi/crm/crm.json

```diff
@@ -595,7 +595,8 @@
                               "type": "string"
                             },
                             "value": {
-                              "type": "string"
+                              "type": "string",
+                              "description": "Most values are passed as string. In the case of lists and tags you can add a '[' and ']' surrounding the values. In the case of geopoints, '[]' surrounding two numeric values (lat,long). In the case of currency values, this value accepts the format '{ currencyCode : \"USD\", amount : 000 } where amount is a numeric value."
```

Comment:
Accepting a struct for currency values is ok but we should also consider accepting a sting and parsing it. Just like we do for dates and times. Golang has a [built in package](https://pkg.go.dev/golang.org/x/text/currency#ParseISO) that gives you the amount and currency code.